utils/src/main/java/se/fortnox/reactivewizard/util/Tuple.java

```diff
@@ -0,0 +1,25 @@
+package se.fortnox.reactivewizard.util;
+
+/**
+ * Generic helper for dealing with Tuples
+ * @param <A> what ever you need!
+ * @param <B> what ever you need!
+ */
+public class Tuple<A,B> {
+
+    private final A value1;
```

Comment:
Because both value1 and value 2 is final and the class is immutable, maybe its redundant to have getters at all? Maybe just have them public?

One could then write 
t.value1 
instead of
t.getValue1()

But maybe we have some static code check that will complain ( even if it would be safe )

Reply:
I think that the Tuple would be more useful with setters, so that it can mutate, but I didn't need that right now. I would rather add setters than go for using the fields.